docs/api/spec/query-api.yaml

```diff
@@ -0,0 +1,306 @@
+openapi: 3.0.0
+info:
+  title: Treetracker Wallet API
+  contact: {}
+  version: '1.28.0'
+servers:
+  - url: https://localhost:3010
+    variables:
+      environment:
+        default: dev
+        enum:
+          - dev     # Development server
+          - test    # Test server
+paths:
+  '/like/types/{type_id}/things/{object_id}':
+    get:
```

Comment:
shouldn't be `post`? because this is something that changes the database?

so you can re-organize the route like this:

`GET /types/{typeid}/things/{thing_id}` to get count
`POST /types/{typeid}/things/{thing_id}/like` to like
 `POST /types/{typeid}/things/{thing_id}/unlike` to unlike